x-pack/plugins/security_solution/common/api/timeline/model/error_schema.mock.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { ErrorSchema } from './error_schema';
+
+export const getErrorSchemaMock = (
```

Comment:
Where's this used?

Reply:
Oh, there was a wrong import. It is used in adjacent `error_schema.test.ts`.